### DISCUSSION TITLE
Feature-Request: Add option to show last change

### DISCUSSION THREAD

**Discussion by jankete** [gerardag/person-entity-card#18] [Ideas]
HI,

would it be possible to add an option (2nd row) to show the last state change? 
So I could display **since when** someone is away/at home/... .

Thanks,
Jankete

**Comment by gerardag** [comment]
Hi,

I think its a good idea. I need to figure it out how to get the information of how much time has been at the same place.

I'm start working on it.

Thanks,
Gerard.

**Comment by ConTS10** [comment]
Hi,
Why does the card not show "Home" as the zone someone is in, even when "showAtHome: true" is set?
I think it looks incomplete this way.

![image](https://user-images.githubusercontent.com/109629356/234593322-57fcb625-f923-4f9a-811e-6c97f1a40189.png)

**Reply by jankete** [reply]
I think this came with one of the last HA updates. Maybe they changed something, because it definitely was displayed correctly earlier this year.

See #35
